rt1_demo_app:
  ros__parameters:
    interbotix_arm:
      kinematics_solver: lma_kinematics_plugin/LMAKinematicsPlugin
      kinematics_solver_search_resolution: 0.005
      kinematics_solver_timeout: 0.1
      kinematics_solver_attempts: 3
      # position_only_ik: true
    interbotix_gripper:
      kinematics_solver_search_resolution: 0.005
      kinematics_solver_timeout: 0.05
      kinematics_solver_attempts: 3
